scripting/include/srccoop_api/classdef.inc

```diff
@@ -9,55 +9,46 @@
 #include <sdkhooks>
 #include <sdktools>
 #include <sourcescramble>
+#include <srccoop_api/color>
```

Comment:
My plan was to eventually split up `utils.inc` and `typedef_game.inc` into multiple files under `srccoop_api` but I'm going to leave that for a new PR. As for `color.inc`, I think it would probably best to leave it for encapsulation incase we decide to expand further (HSL implementation, ect.).

Reply:
So was my plan too, already have some encapsulated classes under the util folder (what I mean)

Reply:
Ah I misunderstood. Moved `color.inc` into the `util` folder.